test/run_tests.pl

```diff
@@ -11,7 +11,7 @@
 
 # Recognise VERBOSE and V which is common on other projects.
 BEGIN {
-    $ENV{HARNESS_VERBOSE} = "yes" if $ENV{VERBOSE} || $ENV{V};
+    $ENV{HARNESS_VERBOSE} = $ENV{V} ? $ENV{V} : $ENV{VERBOSE}
```

Comment:
An even easier way to express that is `$ENV{HARNESS_VERBOSE} = $ENV{V} // $ENV{VERBOSE};`

Reply:
Cool - I'll use that.